src/otx/algorithms/common/adapters/torch/utils/bs_search_algo.py

```diff
@@ -51,7 +52,31 @@ def _try_batch_size(self, bs: int) -> Tuple[bool, int]:
             else:
                 raise e
 
-        max_memory_allocated = torch.cuda.max_memory_allocated(device=None)
+        max_memory_allocated = torch.cuda.max_memory_reserved(device=None)
```

Comment:
```suggestion
        max_memory_reserved = torch.cuda.max_memory_reserved(device=None)
```
Changing the variable name is perferable

Reply:
all names are changed at [3c88d39](https://github.com/openvinotoolkit/training_extensions/pull/2533/commits/3c88d3930cea82a0c5645557f31675f064ef5e12)